index.ts

```diff
@@ -1,4 +1,4 @@
-/// <reference path='typings/tsd.d.ts' />
+/// <reference path='typings/definitions.d.ts' />
```

Comment:
The intent is for each `.ts` file to only reference `typings/definitions.d.ts` for stability.  Having it depend upon the path prefix `lib/` would require the reference to change if `index.ts` moved.